Makefile

```diff
@@ -306,22 +306,26 @@ serve:
 	 @echo "INFO     Starting a server at http://localhost:8000/"
 	 @cd build/website && python3 -m http.server 8000
 
+.PHONY: upload
```

Comment:
Put the phony at the right place, right now upload nd upload-mirror are switched. (Just an optical thing)

Reply:
Fixed.